common/utilities/os/h_result.h

```diff
@@ -0,0 +1,56 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include "../../src/types.h"
```

Comment:
Not good...
What dependencies do we have there?
Can we remove them?

Reply:
its needed for to_string() and for librealsense::windows_backend_exception

Reply:
Removed dependencies on types.h